test/options.test.js

```diff
@@ -67,6 +67,33 @@ exports.mappingOverSomeAppliesFunctionToValue = function(test) {
     test.done();
 };
 
+
+exports.flatMapOnNoneToNone = function(test) {
```

Comment:
Ok, I will change later, thanks :)

On Tuesday, 5 January 2016, Michael Williamson <notifications@github.com>
wrote:

> In test/options.test.js
> <https://github.com/mwilliamson/node-options/pull/8#discussion_r48879985>:
>
> > @@ -67,6 +67,33 @@ exports.mappingOverSomeAppliesFunctionToValue = function(test) {
> >      test.done();
> >  };
> >
> > +
> > +exports.flatMapOnNoneToNone = function(test) {
>
> I'm suggesting that the existing test names aren't especially clear, and
> that writing in the style of the example I wrote above might be clearer.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mwilliamson/node-options/pull/8/files#r48879985>.
>


-- 
*Kfir Bloch*

Cell: 054-568-5848

40 Hanamal street, Tel Aviv, Israel


*"At some point you are going to lose consistency"*